.CI/create_feedstocks

```diff
@@ -7,21 +7,32 @@ set -e
 # recipes from. Currently this is `master`.
 git checkout "${TRAVIS_BRANCH}"
 
-# Download and setup Miniconda
-mkdir -p ~/miniconda_staging
-pushd ~/miniconda_staging
-curl -L -O https://raw.githubusercontent.com/conda-forge/conda-smithy/e976c7e84bb3c4846e7562afd1a42c4b535b51f5/bootstrap-obvious-ci-and-miniconda.py
-python bootstrap-obvious-ci-and-miniconda.py ~/miniconda x64 3 --without-obvci && source ~/miniconda/bin/activate root
-popd
-rm -rf ~/miniconda_staging
+# Install Miniconda.
+echo ""
+echo "Installing a fresh version of Miniconda."
+[ ! -f ~/cache/miniconda.sh ] && curl -L https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh > ~/cache/miniconda.sh
+bash ~/cache/miniconda.sh -b -p ~/miniconda
+source ~/miniconda/bin/activate root
 
+# Reuse packages from the cache.
+mkdir -p ~/cache/pkgs
+rm -rf ~/miniconda/pkgs
+mv ~/cache/pkgs ~/miniconda/pkgs
```

Comment:
We could try a different approach

Simply define the build cache dir in the conda config so we dont need to move and copy stuff around.

1.) Install conda
2.) Set the package cache to a different place where we set this in the cached

Not sure if conda will complain if the folder already exists :-| ?